riptable/rt_str.py

```diff
@@ -915,5 +917,81 @@ def char(self, position: Union[int, List[int], np.ndarray]):
     nb_char_par = _njit_par(_nb_char)
 
 
+MIN_INT = np.iinfo(np.int32).min

```

Comment:
Any reason not to reuse riptable.rt_enum.INVALID_DICT or riptable.rt_enum.INVALID_POINTER_32? If the invalid value conventions change then CatString does as well

Reply:
No, good idea.